.rubocop

```diff
@@ -0,0 +1,5 @@
+--parallel
+--display-cop-names
+--extra-details
+--display-style-guide
+--display-time
```

Comment:
Maybe this one is just me, but I tend to find reporting of how long steps take useful.